### PR TITLE
GUI/map/improve default view

### PR DESCRIPTION
**Pull request by osvegn**
Update assets
Set limit to the map's movements
Set key `G` to reset the map to the default view
Set the view in the center of the map at the beginning
Add images in the inventory menu

### CODE REVIEW

File: src/zappy_gui_src/Core/SFML/Map/Map.cpp [hunk 1]
```diff
@@ -54,6 +54,17 @@ void Map::_initRessourcesPaths()
 
 }
 
+void Map::_SetDefaultMapOrigin()
+{
+    sf::Vector2f origin;
+
+    origin.x = -(int(_window.get()->getSize().x) / 2);
+    origin.y = (((_tile.back().get()->getGlobalBound().top + _tile.back().get()->getGlobalBound().height) - _tile.front().get()->getGlobalBound().top) / 2 - _window.get()->getSize().y / 2);
```
Comment (pariciadra) [inline src/zappy_gui_src/Core/SFML/Map/Map.cpp:62]: Should i trust or ?

File: src/zappy_gui_src/Core/SFML/Map/Map.cpp [hunk 2]
```diff
@@ -106,9 +119,18 @@ void Map::_updateMoveMap()
...
         moveMap.x = -value;
-    for (auto &it : _tile) {
-        if (moveMap.x || moveMap.y)
+    if (moveMap.x < 0 && _tile.at((_mapSize.y - 1) * _mapSize.x).get()->getGlobalBound().left > _window.get()->getSize().x / 2)
+        return;
+    if (moveMap.x > 0 && _tile.at((_mapSize.x - 1)).get()->getGlobalBound().left + _tile.at((_mapSize.x - 1)).get()->getGlobalBound().width < _window.get()->getSize().x / 2)
+        return;
+    if (moveMap.y < 0 && _tile.at(0).get()->getGlobalBound().top > _window.get()->getSize().y / 2)
+        return;
+    if (moveMap.y > 0 && _tile.at(_mapSize.x * _mapSize.y - 1).get()->getGlobalBound().top + _tile.at(_mapSize.x * _mapSize.y - 1).get()->getGlobalBound().height < _window.get()->getSize().y / 2)
```
Comment (pariciadra) [inline src/zappy_gui_src/Core/SFML/Map/Map.cpp:128]: Same

**Review by pariciadra — COMMENTED** [review]
Idk what to say about the 240 char of compute except trust you, maybe add some comment to explain all these values are next Time

**Review by LucasTesnier — APPROVED** [review]
NTR!

File: src/zappy_gui_src/Core/SFML/Map/Map.cpp (hunk 1 above)
Comment (osvegn) [reply src/zappy_gui_src/Core/SFML/Map/Map.cpp:62]: yes

File: src/zappy_gui_src/Core/SFML/Map/Map.cpp (hunk 2 above)
Comment (osvegn) [reply src/zappy_gui_src/Core/SFML/Map/Map.cpp:128]: yes

**Review by GuyomT — APPROVED** [review]
NTR

**Pull request merged**